modin/config/pubsub.py

```diff
@@ -48,7 +48,7 @@ class ExactStr(str):
 _TYPE_PARAMS = {
     str: TypeDescriptor(
         decode=lambda value: value.strip().title(),
-        normalize=lambda value: value.strip().title(),
+        normalize=lambda value: str(value).strip().title(),
```

Comment:
`value` is expected to already be `str` here tbh...

Reply:
It is expected, but it looks like `object` is not enough for `mypy` to be satisfied. I wasn't sure how to get around this.

Reply:
I see the input type of `normalize` in `normalize: Callable[[object], object]`. Is that why?

Reply:
I guess a cop-out would be to do something like `Callable[[Any], object]`